sdks/typescript/featurehub-javascript-client-sdk/app/context_impl.ts

```diff
@@ -211,3 +218,20 @@ export class ClientEvalFeatureContext extends BaseClientContext {
   }
 
 }
+
+export class LocalClientContext extends BaseClientContext {
+  constructor(environment: Environment) {
```

Comment:
could you please move this to its own file and add it to the index.ts as an export? The import of the ClientFeatureRepository class is breaking the separation of concerns here.

Reply:
Sure. What about `sdks/typescript/featurehub-repository/app/client_context.ts`? Should I split it too?

Reply:
client_context.ts only has the one interface? I think in this case - if we have one class that needs the actual repository, and thus "breaks the contract" its going to be cognitively messy if we don't break the Server & Client implementations into their own files as well, otherwise people will be confused as to where to find them. As such, perhaps leave the LocalClientContext where it is for now and I can do the refactoring later if it really irks me :-)